integration-sqs-to-lambda/snippet-data.json

```diff
@@ -102,7 +113,7 @@
       "linkedin": "girishbalachandran"
     },
     {
-      "headline": "Python Example Presented by Girish B",
+      "headline": "Python & Ruby Example Presented by Sanskar",
```

Comment:
Did you provide both the python and ruby example @ssnskar ? What happened to Girish B?